app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/ScheduledOverpaymentsClaim.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.models.claim
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.cdsreimbursementclaim.models.eis.declaration.response.BankAccountDetails
+import uk.gov.hmrc.cdsreimbursementclaim.models.ids.MRN
+import uk.gov.hmrc.cdsreimbursementclaim.utils.MapFormat
+
+final case class ScheduledOverpaymentsClaim(
```

Comment:
This has to be the same as `OverpaymentsScheduledJourney.Output` in the frontend

Reply:
@arturopala, I have updated the schema to match output from frontend after pairing with @mghmay

Reply:
@airudah2000 @mghmay I think the container for reimbursement value should be AmountPaidWithCorrect in overpayments

Reply:
And it should be interpreted accordingly